modules/infield_quickstart/cdf_infield_location/README.md

```diff
@@ -0,0 +1,62 @@
+# cdf_infield_location
+
+This module contains location-specific configurations for Infield Quickstart. This is a simplified setup designed for quick deployment.
+
+## Overview
+
+The Infield Quickstart module provides a streamlined configuration for setting up Infield with minimal dependencies. Unlike the full Infield module, this quickstart version:
+
+- Uses a single auth group (normal users) instead of multiple roles
+- Does not include data transformation pipelines
+- Uses a simplified configuration structure
+- Focuses on getting Infield up and running quickly
+
+## Configuration
+
+The module uses the `first_location` variable (set in the parent `default.config.yaml`) to configure location-specific resources. By default, this is set to `oid`.
+
+In [./default.config.yaml](default.config.yaml), you will find configuration values that need to be updated:
+
+- `infield_location_normal_users_source_id`: The source ID from your identity provider for the normal users group
+
+## App Config
+
+This module creates a configuration instance of Infield by populating the APM_Config data model with a configuration. This configuration ties together groups, spaces, and location filters for a specific Infield location.
+
+The configuration is defined in:
+- [InFieldLocationConfig](./cdf_applications/infield_quickstart.InFieldLocationConfig.yaml) - Application configuration with feature toggles
+- [LocationFilter](./locations/infield_quickstart.LocationFilter.yaml) - Location filter defining data models, instance spaces, and views
+
+## Auth
+
+The module creates one group that needs a matching group in the identity provider that the CDF project is configured with:
```

Comment:
I've limited knowledge about Infield but from my knowledge of InRobots you might need an additional 'Infield Admin' group, which can write to / change `APM_config`?